tensorflow/java/src/main/java/org/tensorflow/Tensor.java

```diff
@@ -637,11 +638,25 @@ private static void throwExceptionIfNotByteOfByteArrays(Object array) {
    * represent more than one TensorFlow data type; for example, 'byte' can represent both {@code
    * uint8} and {@code string}, with the latter being the default interpretation.
    */
-  private static DataType dataTypeOf(Object o) {
+  private static Class<?> baseObjType(Object o) {
```

Comment:
Javadoc above needs to be updated.
(Or you can just remove it - 5 line private helper methods don't really need a Javadoc :)